hudi-hadoop-common/src/test/java/org/apache/hudi/common/model/TestHoodieAvroIndexedRecord.java

```diff
@@ -36,21 +39,22 @@ class TestHoodieAvroIndexedRecord {
 
   @Test
   void testIsBuiltInDelete() {
-    Schema schema = SchemaBuilder.record("TestRecord")
-        .fields()
-        .optionalBoolean("_hoodie_is_deleted")
-        .requiredString("field2")
-        .endRecord();
-    GenericRecord record1 = new GenericRecordBuilder(schema)
+    HoodieSchema hoodieSchema = HoodieSchema.createRecord("TestRecord", null, null,
+        Arrays.asList(
+            HoodieSchemaField.of("_hoodie_is_deleted",
+                HoodieSchema.createNullable(HoodieSchema.create(HoodieSchemaType.BOOLEAN)), null, HoodieSchema.NULL_VALUE),
```

Comment:
The createNullable also takes in a HoodieSchemaType to make test code like this less verbose